oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreService.java

```diff
@@ -81,7 +81,6 @@
 import org.apache.jackrabbit.oak.plugins.blob.datastore.SharedDataStoreUtils;
 import org.apache.jackrabbit.oak.plugins.document.persistentCache.PersistentCacheStats;
 import org.apache.jackrabbit.oak.plugins.document.util.MongoConnection;
-import org.apache.jackrabbit.oak.plugins.document.util.SystemPropertySupplier;
```

Comment:
This is an un-intended change, please revert this.

Reply:
I think you forgot to address this change.

Reply:
Yeah, this was ununsed import, and since I made some other changes in this file, the unused import was removed too.